src/v2/Apps/Fair/Components/__tests__/FairBoothRail.jest.tsx

```diff
@@ -0,0 +1,131 @@
+import { setupTestWrapperTL } from "v2/DevTools/setupTestWrapper"
+import { FairBoothRailFragmentContainer } from "../FairBoothRail"
+import { graphql } from "react-relay"
+import { FairBoothRail_Test_Query } from "v2/__generated__/FairBoothRail_Test_Query.graphql"
+import { BoothFilterContextProvider } from "../BoothFilterContext"
+import { fireEvent, screen } from "@testing-library/react"
+import { AnalyticsContext } from "v2/System"
+import { OwnerType } from "@artsy/cohesion"
+import { useTracking } from "v2/System/Analytics/useTracking"
+import { useRouter } from "v2/System/Router/useRouter"
+
+jest.unmock("react-relay")
+jest.mock("v2/System/Analytics/useTracking")
+jest.mock("v2/System/Router/useRouter")
+
+const { renderWithRelay } = setupTestWrapperTL<FairBoothRail_Test_Query>({
+  Component: props => {
+    if (props.show) {
+      return (
+        <AnalyticsContext.Provider
+          value={{
+            contextPageOwnerId: "context-page-owner-id",
+            contextPageOwnerSlug: "context-page-owner-slug",
+            contextPageOwnerType: OwnerType.show,
+          }}
+        >
+          <BoothFilterContextProvider filters={{ sort: "NAME_ASC", page: 2 }}>
+            <FairBoothRailFragmentContainer show={props.show} />
+          </BoothFilterContextProvider>
+        </AnalyticsContext.Provider>
+      )
+    }
+
+    return null
+  },
+  query: graphql`
+    query FairBoothRail_Test_Query @relay_test_operation {
+      show(id: "show-id") {
+        ...FairBoothRail_show
+      }
+    }
+  `,
+})
+
+describe("FairBoothRail", () => {
+  const mockTracking = useTracking as jest.Mock
+  const mockUseRouter = useRouter as jest.Mock
+  const trackEvent = jest.fn()
+
+  beforeEach(() => {
```

Comment:
**suggestion (non-blocking):** Could this be a `beforeAll` call? The tests still pass locally when I make that change. Just wondering!

```suggestion
  beforeAll(() => {
```

Reply:
yep, sure

Reply:
yep, we can replace `beforeEach` with `beforeAll`